gcloud/credentials.py

```diff
@@ -181,7 +187,11 @@ def _get_pem_key(credentials):
         raise TypeError((credentials,
                          'not a valid service account credentials type'))
 
-    return crypto.load_privatekey(crypto.FILETYPE_PEM, pem_text)
+    if crypto is None:
+        raise EnvironmentError('pyOpenSSL must be installed to load a '
```

Comment:
nit: I prefer this

```
if crypto is None:
   raise EnvironmentError(
       'pyOpenSSL must be installed to load a private key')
```

But I don't feel strongly enough to make anyone else do it.